Agents/ddpg.py

```diff
@@ -0,0 +1,268 @@
+import joblib
+import tensorflow as tf
```

Comment:
Importing Tensor at the top of the module is generally a bad practice because it's very slow but we are out of time so we will have to go with it.